config/locales/override/en.yml

```diff
@@ -8,6 +8,38 @@ en:
   #     copyright_html: "&copy; Copyright 2011%{to_date} Northwestern University"
   #     logo_alt: "Northwestern University Logo"
 
+  # models:
```

Comment:
Can you add a comment here explaining why this is here?
Like "In order to change the usage of NetID or Chart string, set the variables in `en.yml` and then make the rest of your changes here" ... or something